src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DataMaskingTests.cs

```diff
@@ -25,12 +25,13 @@ public class DataMaskingTests : SqlTestsBase
     {
         protected override void SetupManagementClients(RestTestFramework.MockContext context)
         {
-            var sqlCSMClient = GetSqlClient();
+            var sqlClient = GetSqlClient(context);
```

Comment:
@jaredmoo I see duplicate code in SetupManagementClients and it's override.
Maybe I am missing something, but if the virtual method creates the same number of clients that the override is creating, any reason to override this function in almost every test class?